release.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+from distutils.version import StrictVersion
+import os
+import re
+import requests
+import subprocess
+import sys
+import tempfile
+
+from client.api import assignment
+from client.utils import auth
+
+GITHUB_TOKEN_FILE = '.github-token'
+GITHUB_REPO = 'Cal-CS-61A-Staff/ok-client'
+OK_SERVER_DOMAIN = 'https://okpy.org'
+
+def abort(message=None):
+    if message:
+        print('ERROR:', message, file=sys.stderr)
+    sys.exit(1)
+
+def shell(command, capture_output=False):
+    kwargs = dict(shell=True, check=True)
+    if capture_output:
+        kwargs.update(stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+    else:
+        print('>', command)
+    try:
+        output = subprocess.run(command, **kwargs)
+    except subprocess.CalledProcessError as e:
+        print(str(e))
```

Comment:
Curious: Any reason this isn't going into sys.stderr? (I'm ambivalent about where it goes)